coap/er-coap-13/er-coap-13.c

```diff
@@ -65,6 +65,7 @@
 /*- Variables -----------------------------------------------------------------------*/
 /*-----------------------------------------------------------------------------------*/
 static uint16_t current_mid = 0;
+uint16_t coap_block_size = COAP_DEFAULT_BLOCK_SIZE;
```

Comment:
This is declared in liblwm2m.h. I think the storage and default value should be set in one of the core files, not here. I would suggest core/packet.c since that is where most of the usage occurs.